### ISSUE TITLE
Hemophage Dormant State slowing you in anti gravity scenarios

### ISSUE THREAD

**Issue by PeriwinkleBuddy** [Skyrat-SS13/Skyrat-TG#17429]
## Round ID:
6929

## Testmerges:
https://github.com/Skyrat-SS13/Skyrat-tg/pull/17290

## Reproduction:
Say your exploring space, shmoving around with your Ion Jetpack, suddenly you realize your running low on blood, as you enter a dormant state, your slowed to a crawl.. in space.. where your jetpack is propelling you.. your slowed to a snails pace.. in space. It just doesnt seem right at all.

Basically, entering Dormant State while in zero gravity slows you down.

**Comment by YakumoChen** [comment]
@GoldenAlpharex
